src/components/ElvaYBPractice2/styles/TopComponent.styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+import BackgroundPic from '../../../assets/ElvaYongbin/BackgroundPic.svg'
+import PatternCirclePic from '../../../assets/ElvaYongbin/images/pattern-circles.svg'
+import SliderIcon from '../../../assets/ElvaYongbin/images/icon-slider.svg'
+
+export const TopComponent = styled.div`
+  background-image: url(${PatternCirclePic});
+  background-repeat: no-repeat;
+  background-position: center;
+  width: auto;
+  height: auto;
+  text-align: center;
+  position: absolute;
+  top: 50px;
+  align-items: center;
+  @media only screen and (max-width: 600px) {
+    margin-top: 30px;
+    height: 22vh;
```

Comment:
can consider setting the height to auto, it should works the same or better

Reply:
Background changes to something else, think it is required.